content/zh-cn/docs/concepts/overview/working-with-objects/kubernetes-objects.md

```diff
@@ -162,21 +177,24 @@ In the `.yaml` file for the Kubernetes object you want to create, you'll need to
 * `metadata` - Data that helps uniquely identify the object, including a `name` string, `UID`, and optional `namespace`
 * `spec` - What state you desire for the object
 -->
-### 必需字段  {#required-fields}
+### 必需字段    {#required-fields}
```

Comment:
Note: there is no specific rule about how many space should it have, next time we can just leave it as is.